components/button/index.css

```diff
@@ -236,6 +287,25 @@ a.spectrum-ActionButton {
 
   cursor: pointer;
   outline: none;
+
+  &:hover {
+    box-shadow: none;
+  }
+  &:active {
+    box-shadow: none;
+  }
+  &.is-selected {
+    &:focus-ring,
+    &.is-focused {
```

Comment:
Does this also need `:focus`? For anything that can physically be focused (i.e. `tabindex` or already focusable), if we ever have `.is-focused`, we need `:focus` on the side, IIRC.

Reply:
Er why was this there at all? It makes it so you can't focus on selected field buttons. Thoughts?

Reply:
TBH I don't think any of those `box-shadow` resets were required... Removing 'em!

Reply:
Most of that was to keep was existing and move it to the correct css file. I'm fine deleting it if box shadows aren't involved anymore and there are no visible problems.

Reply:
Removed! Things look right still.